rotkehlchen/data/importer.py

```diff
@@ -250,6 +250,7 @@ def _consume_cryptocom_entry(self, csv_row: Dict[str, Any], **kwargs: Any) -> No
             'card_cashback_reverted',
             'reimbursement',
             'viban_purchase',
+            'crypto_viban_exchange',
```

Comment:
what the heck is a crypto viban exchange?

Reply:
Was described in the beautiful issue. https://github.com/rotki/rotki/issues/3685 For what I've assumed entries containing viban are related to actions involving going in/out from fiat